pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-eusage-reports</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
Why 1.1.0 rather than 1.0.0?

Reply:
1.1.0 is the next release in master. The current release is 1.0.0.